docs/section10/10-7.md

```diff
@@ -100,21 +98,21 @@
-                            - Check if capacity is not in the <240,000 Btuh and greater or equal to 135,000 Btuh size category, carry on if not: `if total_cool_capacity_b >= 240000:` 
-                                - Get baseline efficiency for the >=240,000 Btuh size category:`COPnf_full_load_240Btuh_b = data_lookup(table_G3_5_2,240Btuh)`
-                                - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_240Btuh_b: meets_lower_or_equiv_size_category_eff_b = TRUE`                   
+                    - Check if capacity is greater or equal to the 135000 Btuh and the efficiency is not equal to a lower size bracket efficiency: `if total_cool_capacity_b >= 135000 AND meets_lower_or_equiv_size_category_eff_b != TRUE:` 
+                        - Get baseline efficiency for the <240,000 Btuh and greater or equal to 135,000 Btuh size category:`COPnf_full_load_135_240Btuh_b = data_lookup(table_G3_5_2,135_240Btuh)`
+                        - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_135_240Btuh_b: meets_lower_or_equiv_size_category_eff_b = TRUE`
+                    - Check if capacity is greater or equal to 240000 size category and the efficiency is not equal to a lower size bracket efficiency: `if total_cool_capacity_b >= 240000 AND meets_lower_or_equiv_size_category_eff_b != TRUE:` 
+                        - Get baseline efficiency for the >=240,000 Btuh size category:`COPnf_full_load_240Btuh_b = data_lookup(table_G3_5_2,240Btuh)`
+                        - Check if the modeled efficiency is consistent with this value, if so set boolean variable to true: `if full_load_efficiency_b == COPnf_full_load_240Btuh_b: meets_lower_or_equiv_size_category_eff_b = TRUE`                   
                                                 
                 **Rule Assertion:**
-                - Case 1: if the modeled efficiency matches the expected baseline efficiency based on the size category or if for systems 3 and 4 the modeled efficiency matches the expected efficiency based on the size category or the efficiency of a smaller size category then pass: `if meets_lower_or_equiv_size_category_eff_b == TRUE: PASS`             
+                - Case 1: if the modeled efficiency matches the expected baseline efficiency based on the size category or if for systems 3 and 4 the modeled efficiency matches the expected efficiency based on the size category or the efficiency of a smaller size category and the efficiency was defined in the correct units then pass: `if meets_lower_or_equiv_size_category_eff_b == TRUE AND was_efficiency_found == TRUE : PASS`             
```

Comment:
Potentially have UNDETERMINED outcome if capacity is missing.

Reply:
Addressed